src/views/TipsList.vue

```diff
@@ -38,12 +38,42 @@
               >
                 {{ $t('views.TipList.SortingHighestRated') }}
               </a>
+              <div
+                v-if="showMenu"
+                class="overlay-dots"
+                @click="showMenu = false"
+              />
+              <div class="actions-menu">
+                <div
+                  class="dots"
+                  :class="{ 'show-menu': showMenu }"
+                  @click.stop="showMenu = true"
+                >
+                  •••
```

Comment:
i think this should be a CSS pseudo `:before` with content or img background

Reply:
Same here, we should apply this change to a unified place I guess